python/paddle/fluid/dataloader/dataset.py

```diff
@@ -222,3 +222,52 @@ def __getitem__(self, idx):
     def __len__(self):
         raise RuntimeError("'{}' should not be called for IterableDataset" \
                 "{}".format('__len__', self.__class__.__name__))
+
+
+class TensorDataset(Dataset):
```

Comment:
can TensorDataset only be used in dygraph? if so,  raise error in static mode

Reply:
Done, thanks!